clang/include/clang/ExtractAPI/Serialization/SymbolGraphSerializer.h

```diff
@@ -36,7 +36,7 @@ using namespace llvm::json;
 /// The Symbol Graph format (https://github.com/apple/swift-docc-symbolkit)
 /// models an API set as a directed graph, where nodes are symbol declarations,
 /// and edges are relationships between the connected symbols.
-class SymbolGraphSerializer : public APISerializer {
+class SymbolGraphSerializer : public APISetVisitor<SymbolGraphSerializer> {
```

Comment:
Renaming would break `CXExtractAPI` because it uses static function `serializeSingleSymbolSGF`

Reply:
Yeah just rename role use.